NuGet.Config

```diff
@@ -4,6 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="dotnet-corert" value="https://dotnet.myget.org/F/dotnet-corert/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
We don't want to take any new feeds in our repo right now. 

Should work with @eerhardt to get these dependencies into our deps feed for the time being.